src/whisper_ctranslate2/whisper_ctranslate2.py

```diff
@@ -185,6 +185,20 @@ def read_command_line():
         "--device", default="auto", help="device to use for CTranslate2 inference"
     )
 
+    parser.add_argument(
+        "--vad_filter",
+        type=bool,
+        default=False,
+        help="Enable the voice activity detection (VAD) to filter out parts of the audio without speech. This step is using the Silero VAD model https://github.com/snakers4/silero-vad.",
+    )
+
+    parser.add_argument(
+        "--vad_min_silence_duration_ms",
+        type=float,
+        default=300,
```

Comment:
Wow, this default value is significantly shorter than the upstream default value (2s), which is unexpected, and it seems to cause the timing of subtitles to be unstable due to my experiences. Can we make this default value stays the same as the upstream one?